### PR TITLE
fixed error of validation in general error handler

### PR DESCRIPTION
**Pull request by FranMedic**

### CODE REVIEW

File: server/middlewares/error.test.js [hunk 1]
```diff
@@ -55,15 +56,20 @@ describe("Given generalErrorHandler function", () => {
   describe("when it receives a ValidationError", () => {
     test("then it should it should invoke res object with a Sended wrong format of request ! (╯°□°）╯︵ ┻━┻, and an error code of 400", () => {
       const res = mockResponse();
-      const ValidateError = () =>
-        new Error("Sended wrong format of request ! (╯°□°）╯︵ ┻━┻");
```
Comment (AlvaroCabero) [inline server/middlewares/error.test.js:59]: Grammar error! *Sent

**Review by AlvaroCabero — CHANGES_REQUESTED** [review]
Check Grammar Mistake in Error Message -line

File: server/middlewares/error.test.js (hunk 1 above)
Comment (FranMedic) [reply server/middlewares/error.test.js:59]: I fixed thank u!!

**Review by AlvaroCabero — APPROVED** [review]
Perfect! Thank u!

**Pull request merged**